documentation/sections/changelog.md

```diff
@@ -1,6 +1,14 @@
 ## Change Log
 
 ```
+releaseHeader('2017-02-15', '1.12.4', '1.12.3')
+```
+
+*   The `cake` commands have been updated, with new `watch` options for most tasks. Clone the [CoffeeScript repo](https://github.com/jashkenas/coffeescript) and run `cake` at the root of the repo to see the options.
```

Comment:
I agree. I thought it was worth mentioning because the docs mentioned the cake commands, though I've been removing those references. They're still mentioned all over the place in the wiki etc.